packages/cashier/src/pages/account-transfer/account-transfer-receipt/account-transfer-receipt.tsx

```diff
@@ -18,9 +46,9 @@ const AccountTransferReceipt = ({
     selected_from,
     selected_to,
     switchAccount,
-}) => {
-    const [is_switch_visible, setIsSwitchVisible] = React.useState(false);
-    const [switch_to, setSwitchTo] = React.useState({});
+}: TAccountTransferReceiptProps) => {
+    const [is_switch_visible, setIsSwitchVisible] = React.useState<boolean>(false);
```

Comment:
@hamid-deriv mmm think that we can remove <boolean> generic because for primary types, we don't need to make the type explicit, since typescript can infer them. but it's just a suggestion